Mathlib/RingTheory/HahnSeries/Binomial.lean

```diff
@@ -0,0 +1,170 @@
+/-
+Copyright (c) 2024 Scott Carnahan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Carnahan
+-/
+import Mathlib.RingTheory.HahnSeries.HEval
+import Mathlib.RingTheory.PowerSeries.Binomial
+
+/-!
+# Binomial expansions of powers of Hahn Series
+We introduce binomial expansions using `embDomain`.
+
+## Main Definitions
+  * `HahnSeries.orderTopSubOnePos` is the group of invertible Hahn series close to 1, i.e., those
+  series such that subtracting one yields a series with strictly positive `orderTop`.
+  * `HahnSeries.binomialFamily`
+
+## Main results
+  * coefficients of powers of binomials
+
+
+-/
+
+open Finset Function
+
+open BigOperators Pointwise
+
+namespace HahnSeries
+
+suppress_compilation
+
+variable {Γ R A : Type*}
+
+variable [LinearOrder Γ] [AddCommMonoid Γ] [IsOrderedCancelAddMonoid Γ] [CommRing R]
+
+namespace SummableFamily
+
+variable [BinomialRing R] [CommRing A] [Algebra R A]
+
+/-- A summable family of Hahn series, whose `n`th term is `Ring.choose r n • (x - 1) ^ n` when
+`x` is close to `1` (more precisely, when `0 < (x - 1).orderTop`), and `0 ^ n` otherwise. These
+terms give a formal expansion of `x^r` as `(1 + (x-1))^r`. -/
```

Comment:
```suggestion
terms give a formal expansion of `x ^ r` as `(1 + (x - 1)) ^ r`. -/
```
I think adding spaces or not in docstrings is up to the author but I think we should at least be consistent in the same docstring.